migration_scripts/sentiment.py

```diff
@@ -0,0 +1,71 @@
+"""Generic batch script for data migration on all three data sources
+If the field sentiment doesn't exist, add a sentiment analysis of the field text to the article, post or tweet.
+"""
+import argparse
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from utils import get_database
+
+def get_arguments():
+    """Get script arguments from the argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description='Script for data migration on sentiment analysis')
+    parser.add_argument('--remote', action='store_true', default=False, help='Data migration on remote database (default: local)')
+    parser.add_argument('--limit', type=int, default=10, help='Limit on modifying entries (default: 10)')
+    subparsers = parser.add_subparsers(dest='data_source', required=True)
+    subparsers.add_parser('rss', help='Reload sentiment analysis on data from RSS feeds')
+    subparsers.add_parser('reddit', help='Reload sentiment analysis on data from reddit')
+    subparsers.add_parser('twitter', help='Reload sentiment analysis on data from twitter')
+    return parser.parse_args()
+
+if __name__ == '__main__':
+    args = get_arguments()
+    print('args:', args)
+
+    database = get_database(args.remote)
+    
+    if args.data_source == 'rss':
+        collection_name = 'rss.articles'
+    if args.data_source == 'reddit':
+        collection_name = 'reddit.posts'
+    if args.data_source == 'reddit':
+        collection_name = 'twitter.tweets'
+
+    collection = database[collection_name]
+
+    # Set up sentiment analyzer
+    analyzer = SentimentIntensityAnalyzer()
+
```

Comment:
Took a look at this part and I'm not sure it this will work as expected. 

Going by the document structures for the respective data sources, this will give us issues for rss articles and reddit posts. An rss post has no field `text` but a field `title` that we then _could_ analyse for sentiment. In the ingestion scripts we chose not to, so that would be up to you/us to do it here now, if we choose to do so.